data_model/src/query.rs

```diff
@@ -1479,19 +1483,23 @@ pub mod transaction {
 }
 
 pub mod block {
-    //! Queries related to `Transaction`.
+    //! Queries related to `Block`.
```

Comment:
```suggestion
    //! Queries related to [`Block`].
```
Or we can't make it to be a link?

Reply:
It doesn't actually involve `Block` directly, same with `transaction` module. I guess it would be better to just say "blocks" and "transactions" there.